.mergify.yml

```diff
@@ -39,6 +39,7 @@ pull_request_rules:
     conditions:
       - author=github-actions[bot]
       - status-success=complete-pr
+      - status-success=complete-push
```

Comment:
Should this be an OR operator? By default, I think it's an AND (I might be wrong; I'll double-check again).

Reply:
The "and" is what I wanted here.

Reply:
I'm OK with it being an AND, personally.

Reply:
Aye, LGTM then. 🙂